cmd/backup-operator/README.md

```diff
@@ -0,0 +1,83 @@
+# etcd backup operator
+
+## Overview
+
+etcd backup operator backups the data of a etcd cluster running on [Kubernetes][Kube] to a remote storage such as AWS [S3][s3].
+
+## Getting Started
+
+Try out etcd backup operator by running it on Kubernetes and then create a `EtcdBackup` Custom Resource which contains the targeting etcd cluster and S3 backup config; the etcd backup operator automatically picks up the `EtcdBackup` Custom Resource, retrieves etcd snapshot, and then saves it to S3.
+>Note: The demo uses the `default` namespace.
+
+Prerequisites: 
+* access to a Kubernetes environment.
+* see [Instruction][etcd_cluster_deploy] to deploy an etcd cluster. 
+
+### Deploy etcd restore operator
+
+Once `example-etcd-cluster` is running, let's create a backup for `example-etcd-cluster` using etcd backup operator. 
+
+First, deploy an etcd backup operator:
+> Note: etcd backup operator also creates EtcdBackup CRD when starting.
+
+```sh
+$ kubectl create -f example/etcd-backup-operator/deployment.yaml
+$ kubectl get pod
+NAME                                    READY     STATUS    RESTARTS   AGE
+etcd-backup-operator-1102130733-hhgt7   1/1       Running   0          3s
+```
+
+### Create AWS Secret
+
+Then, create an Kubernetes secret that contains aws config/credential; etcd operator uses the secret to gain access to S3 in order to save the etcd snapshot.
```

Comment:
>etcd operator uses the secret to gain access to S3 in order to save the etcd snapshot.

the etcd **backup** operator uses the secret to gain access to S3 in order to save the etcd snapshot.

Reply:
Actually I'm going to reference the `Create AWS Secret` section in the restore operator doc as well. Can you move this line just before this section like:
```
The etcd backup operator needs a secret containing the aws config/credentials to access S3 in order to save the etcd backup.

### Create AWS Secret

Verify that the local aws config and credentials files exist:
...

Reply:
`The etcd backup operator needs a secret containing the aws config/credentials to access S3 in order to save the etcd backup.` will end up under `Deploy etcd backup operator` section. Is that what we want?

Reply:
On second thought it's a small section and it's better not to link to another doc. I'll just add the section directly. Please disregard my last comment and don't move it into the previous section. Sorry for the confusion.

Reply:
it's all good.